mappgene/subscripts/ivar.py

```diff
@@ -12,6 +12,8 @@ def run_ivar(params):
     input_reads = params['input_reads']
     variant_frequency = params['variant_frequency']
     read_cutoff_bp = params['read_cutoff_bp']
+    thread_usage = params['threads']
+    trim_front_tail = params['trim_f_t']
```

Comment:
I think it would help to have the parameter with the same name, so 'trim_front_tail' too.